Add Shared Storage DevTools information (#4565)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/shared-storage/index.md

For the Private Aggregation demo, you can preview unique [reach measurement](/docs/privacy-sandbox/shared-storage/unique-reach), [demographics measurement](/docs/privacy-sandbox/shared-storage/user-demographics), and [K-frequency measurement](/docs/privacy-sandbox/shared-storage/k-freq-reach).

### DevTools integration

Starting from Chrome Canary M110, you can view and modify Shared Storage entries in DevTools. 

{% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/GHrD4x12BqA8QOPH44VS.png", alt="Shared Storage section in Chrome DevTools", width="426", height="554" %}

> Under the 'Applications' tab, you can find a list of origins that have Shared Storage entries. You can edit the entry by right clicking on them. The `Budget` value represents the [daily entropy budget](https://github.com/WICG/shared-storage#budgeting) for Fenced Frames top-level navigation. 

Reviewer: ```suggestion
In **Applications**, you can find a list of origins that have Shared Storage entries. Edit the origin by right clicking on the entry. The `Budget` value represents the [daily entropy budget](https://github.com/WICG/shared-storage#budgeting) for Fenced Frames top-level navigation. 
```

Reviewer: Style guide related updates
